app/i18n/en/gen.php

```diff
@@ -198,17 +198,18 @@
 		'mastodon' => 'Mastodon',
 		'movim' => 'Movim',
 		'pinboard' => 'Pinboard',
-		'pinterest' => 'Pinterest',	// TODO
+		'pinterest' => 'Pinterest',
 		'pocket' => 'Pocket',
 		'print' => 'Print',
 		'raindrop' => 'Raindrop.io',
-		'reddit' => 'Reddit',	// TODO
+		'reddit' => 'Reddit',
 		'shaarli' => 'Shaarli',
 		'twitter' => 'Twitter',
 		'wallabag' => 'wallabag v1',
 		'wallabagv2' => 'wallabag v2',
-		'whatsapp' => 'Whatsapp',	// TODO
-		'xing' => 'Xing',	// TODO
+		'web-sharing-api' => 'Default system sharing',
```

Comment:
Another possibility could be something like:
```suggestion
		'web-sharing-api' => 'Built-in sharing',
```